packages/data-context/src/data/ProjectConfigIpc.ts

```diff
@@ -305,7 +305,8 @@ export class ProjectConfigIpc extends EventEmitter {
     // If they've got TypeScript installed, we can use
     // ts-node for CommonJS
     // ts-node/esm for ESM
-    if (hasTypeScriptInstalled(this.projectRoot)) {
+    // If we're in a cy in cy project, we will handle registering typescript directly in the server entry point
+    if (!process.env.CYPRESS_INTERNAL_E2E_TESTING_SELF_PARENT_PROJECT && hasTypeScriptInstalled(this.projectRoot)) {
```

Comment:
Electron must have changed something with their `spawn` call. We're unsure what, but when cy-in-cy all of this logic is unnecessary.